### DISCUSSION TITLE
How to cancel Git LFS subscription?

### DISCUSSION THREAD

**Discussion by AbdullahKasri** [community/community#65860] [New to GitHub]
Hi everyone!

I wanted to ask whether anyone knows how to cancel an LFS subscription.
I've been trying to do so for a while but could not find any way on my profile of billing settings.

Thanks!

**Comment by nirav-gajera** [comment]
https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/downgrading-your-accounts-plan

**Reply by jonathanlocke** [reply]
This page does not explain how to cancel an LFS subscription. It does not even mention LFS!

**Reply by oddFEELING** [reply]
I am in the same boat. Need to downgrade as I don't need it anymore.

**Reply by oddFEELING** [reply]
@jonathanlocke Did you find a solution for this?

**Reply by nauticalist** [reply]
Any solutions?

**Comment by jeorgmartin2** [comment]
You’re not alone GitHub doesn’t make LFS cancellation very obvious. Usually it’s handled through the billing settings at the organization or account level, not the repo itself. Thanks for raising it here; clear guidance like this is as valuable as good recommendations for [Remodeling in Richmond](https://remodellerrichmond.com/) when people are trying to make the right choice.
